cmd/configure.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/speakeasy-api/openapi-generation/v2/pkg/generate"
+	"github.com/speakeasy-api/sdk-gen-config/workflow"
+	"github.com/speakeasy-api/speakeasy/internal/auth"
+	"github.com/speakeasy-api/speakeasy/internal/interactivity"
+	"github.com/speakeasy-api/speakeasy/prompts"
+	"github.com/spf13/cobra"
+)
+
+var configureCmd = &cobra.Command{
+	Use:   "configure",
+	Short: "Configure your Speakeasy SDK Setup.",
+	Long:  `Configure your Speakeasy SDK Setup.`,
+	RunE:  interactivity.InteractiveRunFn("What do you want to configure?"),
+}
+
+var configureSourcesCmd = &cobra.Command{
+	Use:     "sources",
+	Short:   "Guided prompts to configure a new or existing source in your speakeasy workflow.",
+	Long:    "Guided prompts to configure a new or existing source in your speakeasy workflow.",
+	PreRunE: interactivity.GetMissingFlagsPreRun,
+	RunE:    configureSources,
+}
+
+func configureInit() {
+	rootCmd.AddCommand(configureCmd)
+	configureSourcesInit()
+}
+
+func configureSourcesInit() {
+	configureSourcesCmd.Flags().StringP("name", "n", "", "the name of the source to configure")
+
+	configureCmd.AddCommand(configureSourcesCmd)
+}
+
+func configureSources(cmd *cobra.Command, args []string) error {
+	if err := auth.Authenticate(cmd.Context(), false); err != nil {
+		return err
+	}
+
+	name, err := cmd.Flags().GetString("name")
+	if err != nil {
+		return err
+	}
+
+	workingDir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	var workflowFile *workflow.Workflow
+	if workflowFile, _, _ = workflow.Load(workingDir); workflowFile == nil {
```

Comment:
Is it worth logging the error if it's anything other than "file not found"?

Reply:
I don't want this to display to the user I think. Using this command without a workflow file is a perfectly valid use case.